tests/api2/test_261_iscsi_cmd.py

```diff
@@ -49,6 +49,9 @@
 skip_persistent_reservations = pytest.mark.skipif(not pyscsi_has_persistent_reservations,
                                                   reason="PYSCSI does not support persistent reservations")
 
+skip_extended_copy4 = pytest.mark.skipif(not hasattr(pyscsi.pyscsi.scsi.SCSI, 'extendedcopy4'),
```

Comment:
Why do we have to support different versions of pyscsi? I don't like the idea of skipping tests because the Jenkins pipeline will be green despite some tests not being ran but I might be missing something.

Reply:
Jenkins **shouldn't** skip anything as I have recently added extendedcopy4 support to the pyscsi '[tester](https://github.com/truenas/python-scsi/tree/tester)' branch being used to run our tests.  However, despite raising a PR ([#131](https://github.com/python-scsi/python-scsi/pull/131)) against upstream several weeks ago I've had zero response so far.  (BTW I consulted @yocalebo & @anodos325 and verified that they agreed that I should proceed with the tests.)

So, the `skip_extended_copy4` is just a nod to the fact that this code is still _somewhat_ in flux.  I can certainly remove it.

Please let me know if you still think it should be removed after reading the above context.